geode-redis/README.md

```diff
@@ -128,210 +128,61 @@ not connected>
-|                    	|                                                    	| XTRIM                         	|
-|                    	|                                                    	| ZADD                          	|
-|                    	|                                                    	| ZCARD                         	|
-|                    	|                                                    	| ZCOUNT                        	|
-|                    	|                                                    	| ZINCRBY                       	|
-|                    	|                                                    	| ZINTERSTORE                   	|
-|                    	|                                                    	| ZLEXCOUNT                     	|
-|                    	|                                                    	| ZPOPMAX                       	|
-|                    	|                                                    	| ZPOPMIN                       	|
-|                    	|                                                    	| ZRANGE                        	|
-|                    	|                                                    	| ZRANGEBYLEX                   	|
-|                    	|                                                    	| ZRANGEBYSCORE                 	|
-|                    	|                                                    	| ZRANK                         	|
-|                    	|                                                    	| ZREM                          	|
-|                    	|                                                    	| ZREMRANGEBYLEX                	|
-|                    	|                                                    	| ZREMRANGEBYRANK               	|
-|                    	|                                                    	| ZREMRANGEBYSCORE              	|
-|                    	|                                                    	| ZREVRANGE                     	|
-|                    	|                                                    	| ZREVRANGEBYSCORE              	|
-|                    	|                                                    	| ZREVRANK                      	|
-|                    	|                                                    	| ZSCAN                         	|
-|                    	|                                                    	| ZSCORE                        	|
-|                    	|                                                    	| ZUNIONSTORE                   	|                   	|
-
-**NOTES:**
-
-[1] - UNLINK is implemented as a synonym to DEL and does not unlink asynchronously.
-[2] - SLOWLOG is implemented as a NoOp.
+The Geode APIs compatible with Redis currently implement a subset of the full Redis command set.
+
+#### <a name="redis-command-status"></a> Supported Commands Compatable With Redis [Return to top](#introduction)
```

Comment:
I apologize, I thought I made this comment in the previous review, but it looks like it wasn't made somehow.  Could we make this title `Supported Redis Commands` to be consistent with the docs and since we are talking about actual Redis commands vs. something we created (a decision we made in [this PR](https://github.com/apache/geode/pull/6150).  Then the link name could potentially change from `redis-command-status` to `supported-commands` (there is a reference to this section in a prior section of the README that will also need to change).

Reply:
> Sorry, a couple of additional changes that seemed not to have been saved in the last review. Also, will there be some final review by the docs team and John Martin?

Yeah, docs team. are added as reviewers here.  FYI,  we're starting to make PR's that only include docs changes related to specific corresponding  feature/code change  PR to help with flow.  John didn't indicate any desire to review the PR, but I'll make sure he knows that it's here if he wants to take a look.

Reply:
> I apologize, I thought I made this comment in the previous review, but it looks like it wasn't saved somehow. Could we make this title `Supported Redis Commands` to be consistent with the docs and since we are talking about actual Redis commands vs. something we created (a decision we made in [this PR](https://github.com/apache/geode/pull/6150). Then the link name could potentially change from `redis-command-status` to `supported-commands` (there is a reference to this section in a prior section of the README that will also need to change).

made changes